orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/model/Pipeline.java

```diff
@@ -42,6 +43,16 @@ public Pipeline(
 
   private String pipelineConfigId;
 
+  private List<Artifact> actualArtifacts = new ArrayList<>();
```

Comment:
What does this name mean?

Reply:
Distinct from `expectedArtifacts`.

Reply:
Ah so these are the ones that are received from the trigger?

Reply:
Or do they also include artifacts resolved via the replacement policy?

Reply:
Will add javadoc comment.